ring/token_generator_test.go

```diff
@@ -34,8 +34,3 @@ func TestRandomTokenGenerator_IgnoresOldTokens(t *testing.T) {
 		}
 	}
 }
-
-// GenerateTokens generates numTokens unique, random and sorted tokens for testing purposes.
-func GenerateTokens(tokensCount int, takenTokens []uint32) Tokens {
```

Comment:
[nit] WDYT if we keep this function, and initialize the token generator inside it?
This way the diff in `ring_test.go` would be minimizined.

Reply:
Many tests call that function multiple times, which would then initialize multiple generators. I tried to avoid having too many generators per test, although there may be some tests where there is more than one used.